### PR TITLE
remove direct placement

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

translating pingcap/docs#7400, pingcap/docs#7823

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs#7400
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by xhebox** [comment]
Updated with the latest commit of the upstream PR

File: placement-rules-in-sql.md [hunk 1]
```diff
@@ -19,23 +19,51 @@ Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群
...
 
-要使用 Placement Rules in SQL 特性，你需要在 SQL 语句中指定一个或多个放置选项 (placement option)。可通过*直接放置 (direct placement)* 或*放置策略 (placement policy)* 来指定放置选项。
+指定放置规则，首先需要创建*放置规则 (placement policy)*。
 
-以下示例中，表 `t1` 和 `t2` 的放置规则相同。`t1` 是通过直接放置指定的规则，而 `t2` 是通过放置策略来指定的规则。
+```sql
+CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
+```
+
+然后就可以用 `CREATE TABLE` 或者 `ALTER TABLE` 绑定规则。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:30]:
```suggestion
然后可以使用 `CREATE TABLE` 或者 `ALTER TABLE` 将规则绑定至表或分区表：
```

File: placement-rules-in-sql.md [hunk 2]
```diff
@@ -19,23 +19,51 @@ Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群
...
+ALTER TABLE t2 PLACEMENT POLICY myplacementpolicy;
 ```
 
-为了能够更轻松地管理放置规则，推荐使用放置策略来指定规则。当你通过 [`ALTER PLACEMENT POLICY`](/sql-statements/sql-statement-alter-placement-policy.md) 更改放置策略后，此更改会自动传播至所有数据库对象。
+`PLACEMENT POLICY` 为全局作用域，不与任何数据库表结构相关联。因此，通过 `CREATE TABLE` 指定放置规则时，无需任何额外的权限。
 
-如果你使用直接放置选项，你需要为每个对象（例如表和分区）都单独更改放置规则。
+## 查看放置规则
+
+如果一张表绑定了放置规则，你就可以用 `SHOW CREATE TABLE` 来查看。还可以用 `SHOW CREATE PLACEMENT POLICY` 来查看已经创建的规则。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:42]:
```suggestion
如果一张表绑定了放置规则，你可以用 `SHOW CREATE TABLE` 来查看。还可以用 `SHOW CREATE PLACEMENT POLICY` 来查看已经创建的规则。
```

File: placement-rules-in-sql.md [hunk 3]
```diff
@@ -19,23 +19,51 @@ Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群
...
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin /*T![placement] PLACEMENT POLICY=`myplacementpolicy` */
+1 row in set (0.00 sec)
+tidb> SHOW CREATE PLACEMENT POLICY myplacementpolicy\G
+*************************** 1. row ***************************
+       Policy: myplacementpolicy
+Create Policy: CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1"
+1 row in set (0.00 sec)
+```
+
+`information_schema.tables` 表和 `information_schema.partitions` 表也有一列 `tidb_placement_policy_name`，可用来查找所有绑定了放置规则的对象：
```
Comment (TomShawn) [inline placement-rules-in-sql.md:59]:
```suggestion
`information_schema.tables` 表和 `information_schema.partitions` 表也有一列 `tidb_placement_policy_name`，用于展示所有绑定了放置规则的对象：
```

File: placement-rules-in-sql.md [hunk 4]
```diff
@@ -19,23 +19,51 @@ Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群
...
+
+```sql
+SELECT * FROM information_schema.tables WHERE tidb_placement_policy_name IS NOT NULL;
+SELECT * FROM information_schema.partitions WHERE tidb_placement_policy_name IS NOT NULL;
+```
 
-`PLACEMENT POLICY` 为全局作用域，不与任何数据库表结构相关联。因此，通过 `CREATE TABLE` 指定放置策略时，无需任何额外的权限。
+所有绑定规则的对象都是 *异步调度* 的。可以用 [`SHOW PLACEMENT`](/sql-statements/sql-statement-show-placement.md) 来查看放置规则的调度进程。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:66]:
```suggestion
所有绑定规则的对象都是异步调度的。可以用 [`SHOW PLACEMENT`](/sql-statements/sql-statement-show-placement.md) 来查看放置规则的调度进程。
```

File: sql-statements/sql-statement-alter-placement-policy.md [hunk 5]
```diff
@@ -9,7 +9,14 @@ summary: TiDB 数据库中 ALTER PLACEMENT POLICY 的使用概况。
 >
 > Placement Rules in SQL 是 TiDB 在 v5.3.0 中引入的实验特性，其语法在 GA 前可能会发生变化，还可能存在 bug。如果你知晓潜在的风险，可通过执行 `SET GLOBAL tidb_enable_alter_placement = 1;` 来开启该实验特性。
 
-`ALTER PLACEMENT POLICY` 用于修改已创建的放置策略。此修改会自动更新至所有使用这些放置策略的表和分区。
+`ALTER PLACEMENT POLICY` 用于修改已创建的放置规则。此修改会自动更新至所有使用这些放置规则的表和分区。
+
+`ALTER PLACEMENT POLICY` 会 *完全替换* 之前定义的规则，而不会和之前的规则 *合并*，比如在下面的例子中，`FOLLOWERS=4` 就被 `ALTER PLACEMENT POLICY` 语句覆盖了：
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-placement-policy.md:14]:
```suggestion
`ALTER PLACEMENT POLICY` 会*完全替换*之前定义的规则，而不会和之前的规则*合并*，比如在下面的例子中，`FOLLOWERS=4` 就被 `ALTER PLACEMENT POLICY` 语句覆盖了：
```

File: sql-statements/sql-statement-show-placement-for.md [hunk 6]
```diff
@@ -9,7 +9,13 @@ summary: TiDB 数据库中 SHOW PLACEMENT FOR 的使用概况。
 >
 > Placement Rules in SQL 是 TiDB 在 v5.3.0 中引入的实验特性，其语法在 GA 前可能会发生变化，还可能存在 bug。如果你知晓潜在的风险，可通过执行 `SET GLOBAL tidb_enable_alter_placement = 1;` 来开启该实验特性。
 
-`SHOW PLACEMENT FOR` 用于汇总直接放置 (direct placement) 和放置策略 (placement policy) 中所有的放置选项，并为特定表、数据库或分区以规范形式呈现这些选项信息。
+`SHOW PLACEMENT FOR` 用于汇总所有放置规则 (placement policy) ，并用统一的形式呈现特定表、数据库或分区的信息。
+
+本语句返回结果中的 `Scheduling_State` 列标识了 Placement Driver (PD) 在当前对象上的调度进程，有以下可能结果：
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-placement-for.md:14]:
```suggestion
本语句返回结果中的 `Scheduling_State` 列标识了 Placement Driver (PD) 在当前对象上的调度进度，有以下可能的结果：
```

File: sql-statements/sql-statement-show-placement.md [hunk 7]
```diff
@@ -9,7 +9,13 @@ summary: TiDB 数据库中 SHOW PLACEMENT 的使用概况。
 >
 > Placement Rules in SQL 是 TiDB 在 v5.3.0 中引入的实验特性，其语法在 GA 前可能会发生变化，还可能存在 bug。如果你知晓潜在的风险，可通过执行 `SET GLOBAL tidb_enable_alter_placement = 1;` 来开启该实验特性。
 
-`SHOW PLACEMENT` 汇总了直接放置 (direct placement) 和放置策略 (placement policy) 中的所有放置选项，并以规范的形式呈现这些选项信息。
+`SHOW PLACEMENT` 汇总了所有放置规则 (placement policy) ，并用统一的形式呈现相关信息。
+
+本语句返回结果中的 `Scheduling_State` 列标识了 Placement Driver (PD) 在当前对象上的调度进程，有以下可能结果：
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-placement.md:14]:
```suggestion
本语句返回结果中的 `Scheduling_State` 列标识了 Placement Driver (PD) 在当前对象上的调度进度，有以下可能结果：
```

**Review by TomShawn — COMMENTED** [review]
Questions to be clarified:

1. Placement options - 放置选项：Should this term be globally modified to "placement rules" or "placement policies"?
2. Placement rules - 放置规则：What's the difference between "placement rules" and "placement policies"?
3. Placement policies - 放置策略：The literal translation for "policy" is "策略" instead of "规则". If "placement policies" are translated in the same way as "placement rules", that can be confusing.

Could you please take a look? Thanks! @xhebox @morgo

**Comment by xhebox** [comment]
@TomShawn 

The feature is called placement rules in SQL. But there are only placement policy objects in SQL. These policies themselves are `placement rules(which only exists on PD), in SQL format`. So policy is almost synonymous as rules now. And most users are using `rules`/`policies` at the same time.

So, it is, policies are objects composed of options. You should not modify options. If we really want to unify something, it is policy and rule.

**Comment by xhebox** [comment]
@TomShawn Now it becomes a translation of pingcap/docs#7400 and pingcap/docs#7823

File: placement-rules-in-sql.md [hunk 8]
```diff
@@ -171,9 +210,9 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 
 目前已知 Placement Rules in SQL 实验特性存在以下限制：
 
-* Dumpling 不支持导出放置策略，见 [issue #29371](https://github.com/pingcap/tidb/issues/29371)。
+* Dumpling 不支持导出放置规则，见 [issue #29371](https://github.com/pingcap/tidb/issues/29371)。
 * TiDB 生态工具，包括 Backup & Restore (BR)、TiCDC、TiDB Lightning 和 TiDB Data Migration (DM)，不支持放置规则。
-* 临时表不支持放置选项，直接放置和放置策略均不支持。
+* 临时表不支持放置规则。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:215]: 这里英文写的是“placement options”，不知需要把这里的“规则”改回为“选项”，还是改英文的“option”呢？

File: placement-rules-in-sql.md [hunk 9]
```diff
@@ -171,9 +210,9 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 
 目前已知 Placement Rules in SQL 实验特性存在以下限制：
 
-* Dumpling 不支持导出放置策略，见 [issue #29371](https://github.com/pingcap/tidb/issues/29371)。
+* Dumpling 不支持导出放置规则，见 [issue #29371](https://github.com/pingcap/tidb/issues/29371)。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:213]: 这里英文写的是“Dumpling does not support dumping placement policies.”。不知能否中英用词统一呢？

File: placement-rules-in-sql.md (hunk 8 above)
Comment (xhebox) [reply placement-rules-in-sql.md:215]: 我觉得不改也行, 这两个都是一种意思. 中文更抽象, 英文更具体.

File: placement-rules-in-sql.md [hunk 10]
```diff
@@ -117,25 +148,33 @@ CREATE TABLE t1 (
...
+CREATE TABLE t2 (a INT);  -- 创建表 t2，默认的放置策略 p2 在 t2 上生效。
 
-CREATE TABLE t2 (a INT);  -- 创建表 t2，默认的放置规则 FOLLOWERS=4 在 t2 上生效。
+CREATE TABLE t3 (a INT) POLICY=p1;  -- 创建表 t3。因为语句中已经指定了其他放置规则，默认的 p2 策略在 t3 上不生效。
 
-CREATE TABLE t3 (a INT) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- 创建表 t3。因为语句中已经指定了其他放置规则，默认的 FOLLOWERS=4 规则在 t3 上不生效。
+ALTER DATABASE test POLICY=p3;  -- 再次更改默认的放置规则，此更改不影响已有的表。
 
-ALTER DATABASE test FOLLOWERS=2;  -- 再次更改默认的放置选项，此更改不影响已有的表。
+CREATE TABLE t4 (a INT);  -- 创建表 t4，默认的放置策略 p3 生效。
```
[the remark this answers is not in the document]
Comment (xhebox) [reply placement-rules-in-sql.md:170]: 故意的, p3是策略, 所以这个不改规则.

File: placement-rules-in-sql.md (hunk 9 above)
Comment (xhebox) [reply placement-rules-in-sql.md:214]: 这个就统一了算了.. DONE

File: sql-statements/sql-statement-alter-placement-policy.md [hunk 11]
```diff
@@ -9,7 +9,14 @@ summary: TiDB 数据库中 ALTER PLACEMENT POLICY 的使用概况。
 >
 > Placement Rules in SQL 是 TiDB 在 v5.3.0 中引入的实验特性，其语法在 GA 前可能会发生变化，还可能存在 bug。如果你知晓潜在的风险，可通过执行 `SET GLOBAL tidb_enable_alter_placement = 1;` 来开启该实验特性。
 
-`ALTER PLACEMENT POLICY` 用于修改已创建的放置策略。此修改会自动更新至所有使用这些放置策略的表和分区。
+`ALTER PLACEMENT POLICY` 用于修改已创建的放置策略。此修改会自动更新至所有绑定这些放置策略的表和分区。
+
+`ALTER PLACEMENT POLICY` 会*完全替换*之前定义的规则，而不会和之前的规则*合并*，比如在下面的例子中，`FOLLOWERS=4` 就被 `ALTER PLACEMENT POLICY` 语句覆盖了：
```
Comment (TomShawn) [inline sql-statements/sql-statement-alter-placement-policy.md:14]: 汉字里没有用斜体表示强调的用法
```suggestion
`ALTER PLACEMENT POLICY` 会完全替换之前定义的规则，而不会和之前的规则*合并*，比如在下面的例子中，`FOLLOWERS=4` 就被 `ALTER PLACEMENT POLICY` 语句覆盖了：
```

File: placement-rules-in-sql.md [hunk 12]
```diff
@@ -19,23 +23,51 @@ Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群
 - 把热点数据的 leader 放到高性能的 TiKV 实例上
 - 将冷数据分离到不同的存储中以提高可用性
 
-## 指定放置选项
+## 指定放置规则
+
+指定放置规则，首先需要创建*放置策略 (placement policy)*。
```
Comment (TomShawn) [inline placement-rules-in-sql.md:28]:
```suggestion
指定放置规则，首先需要创建放置策略 (placement policy)。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: placement-rules-in-sql.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (TomShawn) [reply placement-rules-in-sql.md:171]: 这里英文中是 rules

[the remark this answers is not in the document]
Comment (xhebox) [reply placement-rules-in-sql.md:171]: 那我改一下英文

[the remark this answers is not in the document]
Comment (xhebox) [reply placement-rules-in-sql.md:171]: `Creates a table t4 with the default policy p3.` 不对啊就是policy

File: placement-rules-in-sql.md [hunk 13]
```diff
@@ -117,25 +149,33 @@ CREATE TABLE t1 (
...
 
 ```sql
-CREATE TABLE t1 (a INT);  -- 创建表 t1，且未指定放置选项。
+CREATE PLACEMENT POLICY p1 PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- 创建放置策略
+
+CREATE PLACEMENT POLICY p2 FOLLOWERS=4;
+
+CREATE PLACEMENT POLICY p3 FOLLOWERS=2;
+
+CREATE TABLE t1 (a INT);  -- 创建表 t1，且未指定放置规则。
```
Comment (lcwangchao) [inline placement-rules-in-sql.md:161]: 这里改成 `CREATE TABLE test.t1 (a INT);` 是不是表达的更明确一点？

File: placement-rules-in-sql.md [hunk 14]
```diff
@@ -173,7 +213,7 @@ PARTITION BY RANGE( YEAR(purchased) ) (
 
 * Dumpling 不支持导出放置策略，见 [issue #29371](https://github.com/pingcap/tidb/issues/29371)。
```
Comment (lcwangchao) [inline placement-rules-in-sql.md:214]: Dumpling 目前可以支持 placement 导出了

File: placement-rules-in-sql.md [hunk 15]
```diff
@@ -173,7 +213,7 @@ PARTITION BY RANGE( YEAR(purchased) ) (
...
 * TiDB 生态工具，包括 Backup & Restore (BR)、TiCDC、TiDB Lightning 和 TiDB Data Migration (DM)，不支持放置规则。
```
Comment (lcwangchao) [inline placement-rules-in-sql.md:215]: 目前生态工具的情况是，Lightning 目前不支持放置规则。CDC/Binlog 不会同步 placement 到下游数据库。BR 支持 placement （In review：https://github.com/pingcap/tidb/pull/33007）需要修改下当前文档的描述。

之后需要 @IANTHEREAL @leoppro @sunzhaoyang 看下？

File: sql-statements/sql-statement-alter-placement-policy.md [hunk 16]
```diff
@@ -21,11 +28,11 @@ PolicyName ::=
...
-    DirectPlacementOption
-|   PlacementOptionList DirectPlacementOption
-|   PlacementOptionList ',' DirectPlacementOption
+    PlacementOption
+|   PlacementOptionList PlacementOption
+|   PlacementOptionList ',' PlacementOption
 
-DirectPlacementOption ::=
+PlacementOption ::=
     "PRIMARY_REGION" EqOpt stringLit
```
Comment (lcwangchao) [inline sql-statements/sql-statement-alter-placement-policy.md:36]: 之前有用户反馈语法图里面无法区分语法糖的配置选项和高级配置选项，我们可不可以把他们分开，比如：

```
PlacementOptionList ::=
    SugarPlacementOptionList
|   AdvancedPlacementOptionList

....    
```

File: placement-rules-in-sql.md [hunk 17]
```diff
@@ -11,6 +11,10 @@ summary: 了解如何通过 SQL 接口调度表和分区的放置位置。
 
 Placement Rules in SQL 特性用于通过 SQL 接口配置数据在 TiKV 集群中的放置位置。通过该功能，用户可以将表和分区指定部署至不同的地域、机房、机柜、主机。适用场景包括低成本优化数据高可用策略、保证本地的数据副本可用于本地 Stale Read 读取、遵守数据本地要求等。
```
Comment (lcwangchao) [inline placement-rules-in-sql.md:12]: 想 comment 的地方被折叠了，只能先放到这里。。。Placement 在 6.0 正式 GA ，所以需要把实验特性的警告去掉。 `tidb_enable_alter_placement` 也已经被移除了

Comment (xhebox) [reply placement-rules-in-sql.md:12]: 这个在别的pr里

File: placement-rules-in-sql.md (hunk 13 above)
Comment (xhebox) [reply placement-rules-in-sql.md:161]: 我觉得没啥必要..

File: information-schema/information-schema-placement-rules.md [hunk 18]
```diff
@@ -5,7 +5,7 @@ summary: 了解 information_schema 表 `PLACEMENT_RULES`。
 
 # PLACEMENT_RULES
 
-`PLACEMENT_RULES` 表展示所有已显式配置的 [Placement Rules in SQL](/placement-rules-in-sql.md) 信息，包括通过放置策略 (placement policy) 和通过直接放置 (directly placement) 进行配置的信息。
+`PLACEMENT_RULES` 表展示所有已显式配置的 [Placement Rules in SQL](/placement-rules-in-sql.md) 信息，即所有放置策略 (placement policy) 的信息。
```
[the remark this answers is not in the document]
Comment (xhebox) [reply information-schema/information-schema-placement-rules.md:8]: 这几个问题要不换个pr吧, 这个pr已经是两个英文pr的对应合并了..

**Review by lcwangchao — APPROVED** [review]

File: sql-statements/sql-statement-alter-placement-policy.md (hunk 16 above)
Comment (xhebox) [reply sql-statements/sql-statement-alter-placement-policy.md:36]: refer pingcap/docs#7885

File: placement-rules-in-sql.md (hunk 14 above)
Comment (xhebox) [reply placement-rules-in-sql.md:214]: refer pingcap/docs#7885

File: information-schema/information-schema-placement-rules.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (xhebox) [reply information-schema/information-schema-placement-rules.md:8]: refer pingcap/docs#7885

**Pull request merged**